lua/terrortown/entities/roles/jester/winstates.lua

```diff
@@ -94,18 +94,23 @@ end
 --Player spawns within three seconds with a random opposite role of the killer
 function JesterWinstateOne(ply, killer)
 	local rd = killer:GetSubRoleData()
-	local avoidedRoles = {}
-
-	for _, v in pairs(roles.GetList()) do
-		if v.defaultTeam == rd.defaultTeam then
-			avoidedRoles[v] = true
+	local reviveRoleCandidates = GetSelectableRoles()
+	local reviveRoles = {}
+
+	-- make sure innocent and traitor are revive candidate roles
+	reviveRoleCandidates[INNOCENT] = reviveRoleCandidates[INNOCENT] or 1
+	reviveRoleCandidates[TRAITOR] = reviveRoleCandidates[TRAITOR] or 1
+	--remove jester from the revive candidate roles
+	reviveRoleCandidates[JESTER] = nil
+
+	for k, _ in pairs(reviveRoleCandidates) do
```

Comment:
you could remove the comma and the underscore :)

Reply:
just out of curiosity: does this matter performance-wise?

Reply:
Since `_` is a global variable, this can be a bit slower as far as I know. But Alfs notice was mostly for readability